File: src/C#/BehavioralPatterns/Mediator/MultipleExamples/ChainReactionExample/FontEditor.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace MultipleExamples.ChainReactionExample
{
    public class FontEditor : Participant
    {
        public FontEditor(Mediator mediator) : base(mediator)
        {

        }

        public void Enable() => Console.WriteLine("FontEditor is enabled.");

        public void Disable() => Console.WriteLine("FontEditor is disable.");
    }
}

File: src/PHP/BehavioralPatterns/Mediator/ChainReaction/FontEditor.php
<?php

namespace BehavioralPatterns\Mediator\ChainReaction;

class FontEditor extends Participant {
    public function __construct(Mediator $mediator) {
        parent::__construct($mediator);
    }

    public function enable(): void
    {
        echo "FontEditor is enabled." . PHP_EOL;
    }

    public function disable(): void
    {
        echo "FontEditor is disabled." . PHP_EOL;
    }
}